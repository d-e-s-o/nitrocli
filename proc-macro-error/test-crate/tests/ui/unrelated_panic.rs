extern crate test_crate;
use test_crate::make_fn;

make_fn!();

fn main() {}
