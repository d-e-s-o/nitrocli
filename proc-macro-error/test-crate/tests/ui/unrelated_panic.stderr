error: proc macro panicked
 --> $DIR/unrelated_panic.rs:4:1
  |
4 | make_fn!();
  | ^^^^^^^^^^^
  |
  = help: message: unrelated panic test
